### PR TITLE
Replace `AlwaysSample` with `ProbabilitySampler` in router (#1215)

### PR DESCRIPTION
**Pull request by ccamel**
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1348)

### CODE REVIEW

**Comment by life1347** [comment]
@ccamel Thanks for the PR! Due to 1.6 was closed, I would like to merge this PR in 1.7.
Before that, it will be great if you can help us to add `TRACING_SAMPLING_RATE` to helm charts as a configurable setting. (It's ok if you're not available for doing this, I will merge the PR then. Let me know what do you think)

**Comment by ccamel** [comment]
Hi @life1347 thanks for your comment. Sure, no problem. I can try to add the missing configuration to helm charts  (and update the doc too).

**Comment by life1347** [comment]
Thanks! I will review it again once it's ready.

**Comment by life1347** [comment]
Hey @ccamel any update to this PR? I'm fine to merge it if you're busy and cannot make any further changes.

**Comment by ccamel** [comment]
Hi @life1347, sorry I was busy, but I'm about to finalize today, if it's ok for you.

**Comment by life1347** [comment]
@ccamel Sure of course! Thanks for all your effort!

**Comment by ccamel** [comment]
@life1347 I think I'm done. I Hope everything is fine now. Let me know. :smiley:

**Comment by life1347** [comment]
ok, let me review it now!

File: charts/README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Parameter | Description | Default
 `router.svcAddressUpdateTimeout` | The length of update lock expiry time for router to get a service URL returns from executor | `30`
 `router.svcAnnotations` | Annotations for router service | None
 `router.useEncodedPath` | For router to match encoded path. If true, "/foo%2Fbar" will match the path "/{var}"; Otherwise, it will match the path "/foo/bar". | `false`
+`router.traceSamplingRate` | Sample with a rate per time window (traces/second) | `0.5`
```
Comment (life1347) [inline charts/README.md]: We use `ProbabilitySampler` internally, but the description is about rate-limiting sampler. And looks like the rate-limiting sampler is not implemented yet. In order not to block this PR, how about we modify the description for `ProbabilitySampler` in this PR. We can always switch the sampler once it's implemented.
ref: https://opencensus.io/tracing/sampling/probabilistic/
ref: https://github.com/census-instrumentation/opencensus-go/issues/1178

Comment (ccamel) [reply charts/README.md]: Yes, I see your point. So, what description should we have? Something like that?
> Uniformly sample traces with the given probabilistic sampling rate.

Comment (life1347) [reply charts/README.md]: yes, i think that would be ok.

Comment (ccamel) [reply charts/README.md]: I've amended the commit. Should be fine now. Thx.

**Comment by life1347** [comment]
LGTM

**Pull request merged**